exdb/tests/integration_tests.py

```diff
@@ -323,7 +325,7 @@ def test_valid_past_experience_creation_view_submit(self):
         end = now() - timedelta(days=1)
         data = self.get_post_data(start, end)
         data['attendance'] = 1
-        data['subtype'] = self.test_past_subtype.pk
+        data['subtypes'] = self.test_past_subtype.pk
```

Comment:
Although this actually does work, I'd feel better if it was `[self.test_past_subtype.pk]` instead of `self.test_past_subtype.pk`.